pkg/lib/web/sec_headers_middleware.go

```diff
@@ -0,0 +1,44 @@
+package web
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+)
+
+var DefaultStrictCSPDirectives = []string{
+	"default-src 'self'",
+	"object-src 'none'",
+	"base-uri 'none'",
+	"block-all-mixed-content",
+}
+
+// SecHeadersMiddleware sends default security headers according to configuration.
+type SecHeadersMiddleware struct {
+	FrameAncestors []string
```

Comment:
How do we populate this field?

Reply:
Previously, we derive it using OAuth client redirect URIs to support silent token refresh, however that mechanism is most likely non-functional in modern browsers due to strict cookies (and also we're not officially supporting it). Therefore I omitted it for now.

Reply:
Can you put this as comment in the code?